doc/reference/reference_lua/box_events/pub-sub.rst

```diff
@@ -0,0 +1,164 @@
+.. _pubsub:
+
+Pub/sub system and its events
+=============================
+
+Overview
+--------
+
+In Tarantool, :term:`pub/sub <publisher/subscriber>` is a :term:`notification` system for server-side
+:term:`events <event>`.
+
+..  glossary::
+
+    event
+
+        An event is a state change or a system update update that triggers the action of other systems.
+
+    publisher/subscriber
+
+        A publish/subscribe pattern, also shortened as pub/sub, is a design pattern that consists of
+        event producers (publishers), event consumers (subscribers or watchers), and event channel.
+        Publishers communicate with watchers asynchronously by broadcasting events.
+        They sent events to the pub/sub system.
+        Then the system delivers received events to all the watchers that react to them.
+
+    notification
+
+        A notification is a message created by one side to inform about the occurrence of the event and describe
+        it to other components.
+
+The pub/sub pattern is associated with subscriptions.
+Each subscription is defined by the certain key.
+
+Built-in events for pub/sub
+---------------------------
+
+Predefined events have a special naming schema -- theirs names always start with the reserved ``box.`` prefix.
+It means that you cannot create new events with it.
+
+The system processes the following events:
+
+*   ``box.id``
+*   ``box.status``
+*   ``box.election``
+*   ``box.schema``
+
+In response to each event, the server sends back certain ``IPROTO`` fields.
+
+The events are available from the beginning as non-:ref:`MP_NIL <box_protocol-notation>`.
+If the events are broadcast before :doc:`box.cfg{} </reference/reference_lua/box_cfg>`,
+then the events values are empty:
+
+..  code-block:: lua
+
+    box.id = {}
+    box.schema = {}
+    box.status = {}
+    box.election = {}
+
+box.id
+~~~~~~
+
+Contains :ref:`identification <box_info_info>` of the instance.
+Value changes are rare.
```

Comment:
I still don't understand when this and other events are generated. I already [asked](https://github.com/tarantool/doc/pull/2858#discussion_r908205067) you to fix this.

Reply:
Added a paragraph about the event generation